src/homework/ProgrammerIvanov.java

```diff
@@ -0,0 +1,18 @@
+package homework;
+
+import java.util.*;
+
+public class ProgrammerIvanov {
+    public static void main(String[] args) {
+        Random rnd = new Random();
+        int seconds = rnd.nextInt(28800), hours = seconds / 60 / 60;
```

Comment:
```suggestion
        int seconds = rnd.nextInt(28800);
        int hours = seconds / 3600;
```
Да, Java позволяет несколько переменных проинициализировать одновременно, но никто так не делает - менее читабельно